package.json

```diff
@@ -11,7 +11,8 @@
     "clear": "docusaurus clear",
     "serve": "docusaurus serve",
     "write-translations": "docusaurus write-translations",
-    "write-heading-ids": "docusaurus write-heading-ids"
+    "write-heading-ids": "docusaurus write-heading-ids",
+    "format": "npx prettier --write ."
```

Comment:
A couple of ideas here:

1. Should we do `prettier --write '**/*.{js,md,mdx,json}'` so we are a bit more clear on what we want to format?
2. Should we add a format-check (`prettier --check '**/*.{js,md,mdx,json}'`)? Could be used on a GH action.